### PR TITLE
Flip buttons and put visibility on new line

### PR DESCRIPTION
**Pull request by abutterworth**
CTAs are left aligned and Primary comes before Secondary

### CODE REVIEW

File: src/components/ProfilePage/elements/VisibilitySelect.messages.jsx [hunk 1]
```diff
@@ -0,0 +1,46 @@
+import { defineMessages } from 'react-intl';
+
+const messages = defineMessages({
```
Comment (albemarle) [inline src/components/ProfilePage/elements/VisibilitySelect.messages.jsx]: Would it make sense to just use the FormControls messages file with a note in the code that they're the same?

File: src/components/ProfilePage/elements/VisibilitySelect.messages.jsx [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+  'profile.formcontrols.who.can.see': {
+    id: 'profile.formcontrols.who.can.see',
```
Comment (albemarle) [inline src/components/ProfilePage/elements/VisibilitySelect.messages.jsx]: This duplicate id will (eventually) be flagged by automated tools.

Comment (abutterworth) [reply src/components/ProfilePage/elements/VisibilitySelect.messages.jsx]: These messages have been removed from the other file. But this should reflect visibility select. Will update

Comment (albemarle) [reply src/components/ProfilePage/elements/VisibilitySelect.messages.jsx]: At least in this case it's just a search and replace, but I am definitely seeing the drawback of that naming convention.  :-P

**Pull request merged**